examples/dataset_endpoint_minio.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from hub.schema.features import Tensor
+import hub
+
+"""Access credentials shown in this example belong to https://play.min.io:9000.
+These credentials are open to public. Feel free to use this service for testing and development. Replace with your own MinIO keys in deployment."""
+
+token = {
+    "aws_access_key_id": "Q3AM3UQ867SPQQA43P2F",
```

Comment:
Even though you received credentials from public services, it's not a good idea to keep them in GitHub, please remove them and let the user define themselves.